File: Java/p10469.java

import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author DarKnight98
 */
public class p10469 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        while (input.hasNext()) {
            int num1 = input.nextInt();
            int num2 = input.nextInt();
            
            String a = Integer.toBinaryString(num1);
            String temp = "";
            for (int i = 0; i < 32-a.length(); i++) {
                temp += "0";
            }
            a = temp + a;
            
            temp = "";
            
            String b = Integer.toBinaryString(num2);
            for (int i = 0; i < 32-b.length(); i++) {
                temp += "0";
            }
            b = temp + b;
            
            int res = 0;
            int pangkat = 0;
            
            for (int i = a.length()-1; i >= 0; i--) {
                if ((a.charAt(i)=='0' && b.charAt(i)=='1') || (a.charAt(i)=='1' && b.charAt(i)=='0')) {
                    res += Math.pow(2, pangkat);
                }
                
                pangkat++;
            }
            
            System.out.println(res);
        }
    }
}

File: C,C++/p10469.cpp
#include <iostream>
#include <bitset>
#include <string>
#include <cmath>

using namespace std;

int main () {
	int num1, num2;
	while (cin >> num1 >> num2) {
		string num1Bit = bitset<32>(num1).to_string();
		string num2Bit = bitset<32>(num2).to_string();
		
		int pangkat = 0;
		int res = 0;
		
		for (int i = num1Bit.length()-1; i >= 0; i--) {
			if (num1Bit.at(i) == '0') {
				if (num2Bit.at(i) == '0') {
					
				} else {
					res += pow(2,pangkat);
				}
			} else {
				if (num2Bit.at(i) == '0') {
					res += pow(2,pangkat);
				} else {
					
				}
			}
			pangkat++;
		}
		
		cout << res << endl;
	}
	return 0;
}